src/acme/ui/CustomerListUI.java

```diff
@@ -155,14 +153,15 @@ public static void main(String[] args) {
 
     // navigation methods-----------------------------
     public void goToEditPage(String customerName) {
-        customerName = customerName.substring(2);
+        //customerName = customerName;
         Customer customerToPass = null;
         for (int x = 0; x < this.customersArray.length; x++) {
-            if (this.customersArray[x].getName().equals(customerName)) {
+            if (this.customersArray[x].getName().equals(customerName)) {                
                 customerToPass = this.customersArray[x];
             }
         }
-        this.getAcmeUI().customerAddUpdate(customerToPass, c -> this.getAcmeUI().customerList());
+        AcmeUI acme = this.getAcmeUI();

```

Comment:
This fixed a stinkin' null pointer problem that took forever to nail down...

Reply:
Yep, thanks Java garbage collection!